operators/hack/deployer/config/plans.yml

```diff
@@ -14,3 +14,14 @@ plans:
     localSsdCount: 1
     nodeCountPerZone: 1
     gcpScopes: https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append
+- id: aks-ci
+  operation: create
+  clusterName: ci
+  provider: aks
+  kubernetesVersion: 1.12.8
```

Comment:
Is there are way to specify version like `1.12`? We had issues with GKE in the past then we specified patch versions. Like after some time we can't start cluster with specified patch version.

Reply:
It requires full version number unfortunately.